xml.html.markdown

```diff
@@ -1,127 +0,0 @@
----
-language: xml
-filename: learnxml.xml
-contributors:
-  - ["João Farias", "https://github.com/JoaoGFarias"]
----
```

Comment:
OMG, I'm sorry. I shouldn't use the web editor carelessly. I'll send a new pull request from my computer.